sample_apps/README.md

```diff
@@ -7,19 +7,19 @@ that demonstrate how to implement the Certifier APIs on different platforms.
   provisioning and using the Certifier Service with simulated enclaves.
   This simple_app can be run on any Linux machine without need for specialized hardware.
 
-- **simple_app_under_gramine/** - This application is similar to the sample program,
-  simple_app/, but one that can be run in Gramine enclaves. The app uses Intel SGX
-  enclaves on Linux platform using the
-  [Gramine library](https://github.com/gramineproject/gramine).
-  A Gramine enclave uses SGX instructions to generate SGX quotes and reports.
-
 - **simple_app_under_oe/** - This is a sample program that can be run on Intel SGX
   enclaves on Linux platform using the
   [Open Enclave SDK](https://openenclave.io/sdk/), referred briefly as 'OE'.
   Unlike the simple_app example program, an OE-application is partitioned into
   "untrusted" and "trusted" portions, also known as the host and enclave parts,
   respectively.
 
+- **simple_app_under_gramine/** - This application is similar to the sample
```

Comment:
No content change. Just re-ordered the bullets, to keep in-step with order of development.